.gitignore

```diff
@@ -4,4 +4,6 @@ node_modules/
 .idea/
 # Sed backup files.
 *-e
+# Vim swap files.
+*.swp
```

Comment:
I don't think this will hurt anything to have there, I mainly put that there as a note so he doesn't have to add to many repos.